src/sidebar.js

```diff
@@ -369,225 +369,243 @@ module.exports = {
-                    },
-                  ],
-                },
-                {
-                  SDK: ["kratos/sdk/overview", "kratos/self-hosted/go"],
-                },
-                "kratos/reference/configuration",
-                "kratos/reference/json-schema-json-paths",
-                "kratos/reference/html-forms",
-                "kratos/reference/jsonnet",
-              ],
-            },
+            "kratos/configuring",
+            "kratos/reference/configuration-editor",
+            "kratos/guides/configuring-cookies",
+            "kratos/guides/multi-domain-cookies",
+            "self-hosted/kratos/configuration/oidc",
+            "self-hosted/kratos/configuration/password",
+            "kratos/guides/setting-up-password-hashing-parameters",
+            "kratos/guides/setting-up-cors",
+            "kratos/guides/production",
+            "kratos/self-hosted/mail-courier-selfhosted",
+            "kratos/self-hosted/mail-courier-templates",
+            "kratos/self-hosted/account-activation-email-verification",
+            "kratos/self-hosted/mfa",
+            "kratos/guides/zero-trust-iap-proxy-identity-access-proxy",
+            "kratos/guides/multi-tenancy-multitenant",
+            "kratos/guides/select-cipher-algorithm",
+            "kratos/guides/secret-key-rotation",
+            "self-hosted/operations/scalability",
+            "kratos/guides/https-tls",
+            "kratos/guides/tracing",
           ],
         },
         {
           type: "category",
-          label: "Ory Hydra",
+          label: "Troubleshooting",
           items: [
-            "hydra/self-hosted/install",
-            "hydra/5min-tutorial",
-            "hydra/self-hosted/deploy-hydra-example",
-            "hydra/self-hosted/upgrade",
-            {
-              type: "category",
-              label: "Configuration",
-              items: [
-                "hydra/reference/configuration",
-                "hydra/reference/configuration-editor",
-                "hydra/self-hosted/configure-deploy",
-                "hydra/self-hosted/dependencies-environment",
-                "hydra/self-hosted/production",
-                "hydra/self-hosted/hsm-support",
-                "hydra/self-hosted/scaling-hydra",
-                "hydra/self-hosted/merge-multiple-db-secrets",
-                "hydra/self-hosted/gitlab",
-                "hydra/self-hosted/secrets-key-rotation",
-                "hydra/self-hosted/kubernetes-helm-chart",
-                "hydra/self-hosted/ssl-https-tls",
-                "hydra/self-hosted/tracing",
-                "hydra/guides/cookies",
-                "hydra/guides/cors",
-                "self-hosted/hydra/debug/csrf",
-              ],
-            },
+            "kratos/debug/troubleshooting",
+            "kratos/debug/csrf",
+            "kratos/debug/performance-out-of-memory-password-hashing-argon2",
+            "kratos/debug/debug-docker-delve-ory-kratos",
```

Comment:
Another topic contained in a single dir. Update filenames to facilitate ordering and turn into an "autogenerated" category?